lib/replicache.dart

```diff
@@ -342,6 +347,23 @@ class Replicache implements ReadTransaction {
     if (onChange != null) {
       scheduleMicrotask(onChange);
     }
+
+    // TODO(arv): We should pass the ref of the current commit to ensure we get
```

Comment:
I don't understand this comment.

Is it saying that with current code, when the underlying db changes we'll start refreshing queries, but the db might be changing underneath, so we might fire subscription A at hash 0, but subscription B at hash 1?

Reply:
I think we should commit to all subscriptions re-fire together at the same hash, otherwise views will update inconsistently. Can fix it in this PR or subsequently, but it should work this way for next milestone.

Reply:
Follow up coming.